src/bridging/providers/across/createAcrossDepositCall.test.ts

```
@@ -0,0 +1,193 @@
+import { createAcrossDepositCall } from './createAcrossDepositCall'
+import { Contract as WeirollContract } from '@weiroll/weiroll.js'
+import { AcrossQuoteResult } from './AcrossBridgeProvider'
+import { QuoteBridgeRequest } from '../../types'
+import { AdditionalTargetChainId, SupportedChainId, TargetChainId } from '../../../chains/types'
+import { CowShedSdk } from '../../../cow-shed'
+import { ACROSS_MATH_CONTRACT_ADDRESSES, ACROSS_SPOOK_CONTRACT_ADDRESSES } from './const/contracts'
+import { OrderKind } from '@cowprotocol/contracts'
+
+const BRIDGE_REQUEST: QuoteBridgeRequest = {
```

ESLint

[error] 'BRIDGE_REQUEST' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)